generate_site/templates/sponsors-base.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  {% include 'head.html' %}
+  <body>
+
+  {% include 'header.html' %}
+
+<main>
+
+  <!-- page cover -->
+  <div class="position-relative overflow-hidden p-1 p-md-3 m-md-1 text-center bg-white">
+    <div class="d-flex w-75 my-md-3 mx-auto align-items-center">
+      <div class="col-10 col-md-10 p-lg-1 mx-auto mt-auto">
+        <h4 class="special-font gradient title-size-2 text-start">Sponsor Prospectus for the first ever PyLadies Global Con!</h4>
+        <h4 class="no-font text-start">Join us online across the globe, Dec 1-3, 2023!</h4>
+      </div>
+      <div class="col-2 col-md-2 d-none d-lg-block">
+        <img class="img-fluid" src="img/sponsors-icon-2.png" />
+      </div>
+    </div>
+  </div>
+
+  <!-- about -->
+
+  <div class="d-md-flex flex-md-equal w-100 mt-md-3 ps-md-3">
+    <div class="bg-white pt-3 pt-md-5 overflow-hidden">
+      <img class="w-100" src="img/about-photo-1.png" />
+    </div>
+    <div class="bg-white pt-3 pt-md-5 overflow-hidden">
+      <img class="w-100" src="img/about-photo-2.png" />
+    </div>
+  </div>
+
+  <div class="d-md-flex flex-md-equal w-100 ps-md-3">
+    <div class="bg-white mr-md-3 pt-3 px-3 pt-md-5 px-md-5 text-left text-white overflow-hidden">
+      <div class="my-3 py-3 text-black" id="the-event">
+        <h2 class="display-8">The Event</h2>
+        <p class="lead">
+          Support PyLadies as we work to improve diversity by bringing fresh
+          talent into the Python community and to support both seasoned and
+          budding developers through workshops, community activities, and local
+          advocacy.
+        </p>
+        <p class="lead">
+          Supporting our first global conference will help PyLadies provide
+          financial support to help more women around the world with training,
+          mentorship, and aid to attend conferences and community events.
+        </p>
+      </div>
+    </div>
+    <div class="bg-pink mr-md-3 pt-3 px-3 pt-md-5 px-md-5 text-left overflow-hidden text-white">
+      <div class="my-3 p-3" id="the-organizers">
+        <h2 class="display-8">To learn more...</h2>
+        <p class="lead">
+          To learn more and to join us as a sponsor, please contact
+          sponsors@pyladies.com, and a member of our sponsorship team will
+          get back to you soon.
+        </p>
+        <p class="lead">
+          <a class="rarrow-link-white text-white" href="#">Get in touch</a>
```

Comment:
Should the href go somewhere? Or to the "sponsors@" email address?

Reply:
yeah...in the meantime I guess the email will do.